### ISSUE TITLE
[TF 2.0 alpha] Keras Callbacks

### ISSUE THREAD

**Issue by snknitin** [tensorflow/tensorflow#27217]
**System information**
- TensorFlow version: 2.0.0-alpha0
- Doc Link:

**When creating a callback , if we need an accuracy threshold for training , previous TF versions have logs.get('acc') but in this version we need to use logs.get('accuracy') for it to work. There wasn't any documentation regarding this change**

**Comment by madebymaze** [comment]
I have had the same issue. Thanks for the pointing this out, @snknitin

**Comment by Saduf2019** [comment]
@snknitin please confirm if the issue still persist

**Comment by MarkDaoust** [comment]
Hi @snknitin, 

Before tensorflow 2.0 keras metrics were sloppy about their naming.

IIRC with TF2.0+ if you say `model.compile(..., metrics=['acc'])` or `model.compile(..., metrics=['accuracy'])` the key used in the logs will match what you specified to the metric argument.

If you want more control over this you can always do `model.compile(..., metrics=[keras.metrics.SparseCategoricalAccuracy(name='my_accuracy')])` and the `name` argument will be used as the log key.

Given that there are 5 upvotes I'll add a note to the tf2 migration guide. mentioning this before I close it.

**Comment by ursu123** [comment]
FYI the documentation still talks about logs.get('acc') fro v2 - https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/Callback

![image](https://user-images.githubusercontent.com/13632815/77125189-bf232900-6a4d-11ea-907b-9cbecca1c32e.png)

**Comment by jvishnuvardhan** [comment]
@snknitin This was resolved in `nightly` docs https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/Callback?version=nightly

I am closing this issue as this was resolved. Please feel free to reopen if you notice this in any other page on TF website. Thanks!
